frontend/packages/dev-console/src/components/import/GiteaIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+const GiteaIcon: React.FC<React.HTMLProps<SVGElement>> = ({ style }): React.ReactElement => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 20 650 587"
+      fill="currentColor"
+      aria-labelledby="icon-title-110"
+      role="img"
+      width="1em"
+      height="1em"
+      style={style}
+    >
+      <title id="icon-title-110">Edit Gitea source code</title>
```

Comment:
bit of a nitpick for this but the alt text for this svg isn't always accurate: e.g., https://github.com/Lucifergene/console/blob/ODC-Git-type-tile-update/frontend/packages/dev-console/src/components/import/git/GitTypeSelector.tsx#L58